packages/suite/src/components/suite/NavigationBar/components/NavigationActions/index.tsx

```diff
@@ -12,7 +12,7 @@ import { NavTor } from './components/NavTor';
 import { NavEarlyAccess } from './components/NavEarlyAccess';
```

Comment:
I find it very annoying, ugly and not working correctly
https://user-images.githubusercontent.com/33235762/158424381-a20c277b-ba67-4c36-918a-4f07366bc778.mov

Reply:
I didn't find nice solution, let's solve it as separate issue when it comes to priorities: https://github.com/trezor/trezor-suite/issues/5162